apps/betterangels-backend/shelters/models.py

```diff
@@ -171,6 +173,7 @@ class Shelter(BaseModel):
     name = models.CharField(max_length=255)
     organization = models.ForeignKey(Organization, on_delete=models.SET_NULL, blank=True, null=True)
     location = PlacesField(blank=True, null=True)
+    geometry = GeometryField(srid=4326, blank=True, null=True)
```

Comment:
https://docs.djangoproject.com/en/5.1/ref/contrib/gis/model-api/#geography 

you'll want to set the field to `geography=true`

Reply:
good point